docs/add-ons/crossplane.md

```diff
@@ -36,6 +38,22 @@ You can optionally customize the Helm chart that deploys `Crossplane` via the fo
   crossplane_irsa_policies = [] # Optional to add additional policies to Crossplane IRSA
 ```
 
+### Crossplane AWS Provider Deployment
+AWS Provider for Crossplane gets deployed by default when you enable `enable_crossplane = true`.
+The below configuration helps you to upgrade the AWS provider version and lets you define custom IAM policies to manage AWS resources through IRSA.
+
+Crossplane requires Admin like permissions to create and update resources similar to Terraform deploy role.
```

Comment:
Since we are showing an S3 example can we scope this down to S3 permissions only? This will demonstrate how to use policies only for services you need.

Reply:
done